anvio/docs/programs/anvi-search-functions.md

```diff
@@ -0,0 +1,33 @@
+This program **searches for keywords in the function annotations of your database.** 
+
+You can use this program to look for specific functon keywords in a %(contigs-db)s, %(genomes-storage-db)s or %(pan-db)s. For example, say you wanted your %(contigs-db)s to search for genes that encoded some type of kinase. You could call 
+
+{{ codestart }}
+anvi-search-functions -c %(contigs-db)s \
+            --search-terms kinase
+{{ codestop }}
+
+By default, the output will be a fairly barren %(functions-txt)s, only telling you which contigs contain genes that matched your search. This will be most helpful as an additional layer in the anvi'o interactive interface, so you can quickly see where the kinase-encoding genes are in the genome. To do this, run anvi-interactive with the `--aditional-layer` parameter with the %(functions-txt)s. 
+
+However, you can also request a much more comprehensive output that contains much more information, including the matching gene's caller id, functional annotation source and full function name. 
+
+For example, to run the same search as above, but with a more comprehensive output, you could call 
+
+{{ codestart }}
+anvi-search-functions -c %(contigs-db)s \
+            --search-terms kinase
+            --full-report kinase_information.txt
+            --include-sequences
+            --verbose
+{{ codestop }}
```

Comment:
most of these code blocks are missing new line escape characters (this one: `\`) at the end :) 

No one will do this, but in theory if someone were to copy paste this line to their terminal and press enter:

```
anvi-search-functions -c %(contigs-db)s \
            --search-terms kinase
            --full-report kinase_information.txt
            --include-sequences
            --verbose
```

It would run each of these lines separately:

```
anvi-search-functions -c %(contigs-db)s --search-terms kinase

--full-report kinase_information.txt

--include-sequences

--verbose
```

But if it looked like this:

```
anvi-search-functions -c %(contigs-db)s \
            --search-terms kinase \
            --full-report kinase_information.txt \
            --include-sequences \
            --verbose
```

Then copy-pasting it to the terminal would have run it this way:

```
anvi-search-functions -c %(contigs-db)s --search-terms kinase --full-report kinase_information.txt  --include-sequences --verbose
```

which is the desired behavior :)

Reply:
Oh whoops! I'll be sure to fix that.